source/agora/common/Amount.d

```diff
@@ -86,6 +86,13 @@ public struct Amount
         return ret;
     }
 
+    /// Non-throwing version of toString
+    public string toStringNT () const @safe nothrow
```

Comment:
Why is this needed? Commit doesn't explain it.

Reply:
> Why is this needed? Commit doesn't explain it.

I had some 'nothrow' function from which I had to print out the 'Amount', I will modify the comment to make it more obvious that this 'non-throwing' version should mainly be used for debugging.